src/engine/Engine.h

```diff
@@ -22,7 +22,7 @@
 
 #include "scheduler/Scheduler.h"
 
-#include "ui/UI.h"
+#include "../ui/UI.h"
```

Comment:
Visual Studio is apparently very stupid and confuses this with `src\game\world\ui\UI.h`
I'm sure there is a better solution than using a relative path here, but it works for now

Reply:
I was thinking about absolute paths, but it would make moving files and directories around more cumbersome. But we'll see.